draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -140,9 +140,50 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 # Security Considerations
 
-TODO Security and Privacy
+## WIMSE ID
+
+The WIMSE ID is scoped within a trust domain and therefore a WIMSE ID is only unique within a trust domain. Using a WIMSE ID without taking into account the trust domain could allow one domain to issue tokens to spoof identities in another domain.  Additionally, the trust domain must be tied to an authorized trusted key through some mechanism such as a JWKS or X.509 certificate chain. The association of a trust domain and a cryptographic trust root MUST be communicated securely out of band.
+
+[TODO: Should there be a DNS name to Trust domain mapping defined?]
+
+## WIMSE ID Token and Proof of Possession
+
+The WIMSE ID token is bound to a secret cryptographic key and is always presented with a proof of possession as described in section TBD. It MUST NOT be used as a bearer token. While this helps reduce the sensitivity of the token it is still possible that a token and its proof of possession may be captured and replayed.  If this risk is mitigated then WIMSE ID tokens may be resued for a period of time with fresh proofs generated for each transaction.  The following are some mitigations for the capture and reuse of the proof of possession (POP):
```

Comment:
Make the first sentence conditional: the ID Token and its PoP are only used in the application-level options, and both are not used in MTLS.